examples/3d/elixir_euler_ec.jl

```diff
@@ -0,0 +1,55 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the compressible ideal GLM-MHD equations
+
+equations = CompressibleEulerEquations3D(1.4)
+
+initial_condition = Trixi.initial_condition_weak_blast_wave
+
+surface_flux = flux_ranocha
+volume_flux  = flux_ranocha
+solver = DGSEM(3, surface_flux, VolumeIntegralFluxDifferencing(volume_flux))
+
+coordinates_min = (-2, -2, -2)
+coordinates_max = ( 2,  2,  2)
+mesh = TreeMesh(coordinates_min, coordinates_max,
+                initial_refinement_level=3,
+                n_cells_max=10_000)
+
+
+semi = SemidiscretizationHyperbolic(mesh, equations, initial_condition, solver)
+
+
+###############################################################################
+# ODE solvers, callbacks etc.
+
+tspan = (0.0, 0.4)
+ode = semidiscretize(semi, tspan)
+
+summary_callback = SummaryCallback()
+
+stepsize_callback = StepsizeCallback(cfl=1.4)
+
+save_solution = SaveSolutionCallback(interval=10,
+                                     save_initial_solution=true,
+                                     save_final_solution=true,
+                                     solution_variables=:primitive)
+
+analysis_interval = 100
+alive_callback = AliveCallback(analysis_interval=analysis_interval)
+analysis_callback = AnalysisCallback(semi, interval=analysis_interval)
+
+callbacks = CallbackSet(summary_callback, stepsize_callback,
+                        save_solution,
+                        analysis_callback, alive_callback)
```

Comment:
You missed the setting `restart_interval = 10` in `examples/3d/parameters_euler_ec.toml`. Hence, you should also create a `SaveRestartCallback`, e.g. as in `examples/3d/elixir_euler_source_terms.jl`.